server/controllers/medical/patients/merge.js

```diff
@@ -34,17 +34,19 @@ router.get('/count_employees', async (req, res, next) => {
 });
 
 router.get('/duplicates', async (req, res, next) => {
-  const sensitivity = req.params.sensitivity || 2;
+  const sensitivity = req.query.sensitivity || 2;
+  const limit = parseInt(req.query.limit, 10) || 25;
```

Comment:
I am going to make a confession, this is the very first time that I discover this functionality, I have the following concern how do you define the limit of values

Reply:
![image](https://user-images.githubusercontent.com/8035588/101906747-c0871980-3bb9-11eb-83a8-939d9ddcd5d1.png)

I have the impression that it does not use the filter defined on the registry

Reply:
This value of 'limit' refers to the number of potential duplicate patients that is shown in the modal (before going back to the patient registry grid).  The user then selects one and then those patients are shown in the main patient registry grid (with the limit filter that can be set in the "Search" modal).  

But the two limits are unrelated.   The number of potential duplicate patients shown in its modal probably should have a separate 'limit' value.   Currently it is hard-coded.  We could, in the future, provide a way to set it.  But I think we can ignore that for now.

I did figure out how to pass in the current patient registry grid 'limit' to the duplicate patients function, but I don't think it makes sense to do.

@jniles What do you think?

Reply:
Yes, @jmcameron is right - this "limit" is not related to the patient shown in the registry.  That URL is `/patients`, while this one is `/patients/duplciates`.

The point of the limit was to make sure we don't overwhelm either the user or the server with large queries.  So far, that has not happened, but it could.  Thanks for fixing it @jmcameron .

Reply:
> This value of 'limit' refers to the number of potential duplicate patients that is shown in the modal (before going back to the patient registry grid). The user then selects one and then those patients are shown in the main patient registry grid (with the limit filter that can be set in the "Search" modal).
> 
> But the two limits are unrelated. The number of potential duplicate patients shown in its modal probably should have a separate 'limit' value. Currently it is hard-coded. We could, in the future, provide a way to set it. But I think we can ignore that for now.
> 
> I did figure out how to pass in the current patient registry grid 'limit' to the duplicate patients function, but I don't think it makes sense to do.
> 
> @jniles What do you think?

No problem